### PR TITLE
New subscription changes

### PR DESCRIPTION
**Pull request by necoline**
- removed `plan-billing` sub-directory
- updated language around subscription
- updated old images

### CODE REVIEW

File: packages/projects-docs/pages/learn/plans/plan-overview.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Pro
+Pro offers unlimited collaboration, umlimited sandboxes and repositories (both private and public), higher VM specs and [much more](https://codesandbox.io/pricing).
+
+
+
+### Creating a new Pro worksapce
+
+1. Go to the [Dashboard](https://codesandbox.io/dashboard).
+1. Open the workspace selector and click on the `Create a Pro workspace` button at the bottom.
```
Comment (alexnm) [inline packages/projects-docs/pages/learn/plans/plan-overview.mdx:21]: Shouldn't we add here the Pro page selection. step 3 would be `Choose to start a trial or create a pro team based on your availability` or something like that

File: packages/projects-docs/pages/learn/plans/_meta.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "plan-overview": "Choosing a plan",
+  "permissions": "User Permissions",
+  "preferences": "Preferences",
+  "workspace": "Workspaces",
```
Comment (alexnm) [inline packages/projects-docs/pages/learn/plans/_meta.json:5]: shouldn't workspaces and pricing faq be a bit higher in the list? User permissions and Preferences are a bit out of place imo, right after choosing a plan

File: packages/projects-docs/pages/learn/plans/workspace.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  A team makes it easier to collaborate on code with others. In a team you share
+  ownership of repositories, sandboxes and templates.
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# Workspaces
+![Workspaces](../images/workspace-selector.png)
+
+CodeSandbox uses **Workspaces** to provide personal and collaborative experiences. You can think of workspaces as the boundaries for collaboration on CodeSandbox: use a [personal workspace](#personal-workspace) for public, individual development work, and use a [Pro workspace](#pro-workspace) for any type of work requiring privacy or collaboration with others.
```
Comment (alexnm) [inline packages/projects-docs/pages/learn/plans/workspace.mdx:14]: personal `or` collaborative?

File: packages/projects-docs/pages/learn/plans/workspace.mdx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+Every sandbox belongs to a workspace, and every branch of a repository has a separate development environment in each workspace. If you are working on something in your personal workspace and decide to collaborate with others, it may be necessary to transfer (in the case of a sandbox) or re-import (in the case of a repository) your work to a Pro workspace.
+
+## Personal Workspace
+
+By default, when you set up an account, you have a free personal workspace. This is a space to create, prototype and build anything you like. You can think of this workspace as your local development transferred to the cloud. The personal workspace contains any public sandboxes and repositories you added while active in your workspace.
```
Comment (alexnm) [inline packages/projects-docs/pages/learn/plans/workspace.mdx:20]: Not sure if the comparison with the local development is right here. Because you still get one VM per sandbox / repo branch, so it's more then your local development environment in a way

File: packages/projects-docs/pages/learn/plans/workspace.mdx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+<Callout emoji="⚠️">
+Free teams have been depricated. Existing free team subscriptions will continue to be supported in their current form.  
+</Callout>
+
+### Collaboration in workspaces
+
+#### Sharing Sandboxes
+
+You can choose which workspace you're in from the team drop-down at the top-left on
```
Comment (alexnm) [inline packages/projects-docs/pages/learn/plans/workspace.mdx:93]: this paragraph feels a bit long and hard to follow. also not sure how it related to sharing sandboxes, vs sandbox management?

**Review by alexnm — APPROVED** [review]
Summary of pull request review comments:

- Add Pro page selection in step 3 of the plan overview.
- Consider reordering the list to prioritize workspaces and pricing FAQ.
- Clarify whether it is personal or collaborative.
- Review the comparison with local development.
- Simplify and clarify the paragraph about sandbox sharing and management.

File: packages/projects-docs/pages/learn/plans/workspace.mdx (hunk 3 above)
Comment (necoline) [reply packages/projects-docs/pages/learn/plans/workspace.mdx:14]: hmm, I think this whole section needs more revision since it reinforces that personal is for individual work and pro is for collab

**Review by alexnm — APPROVED** [review]
